lib/rules/number-max-precision/index.js

```diff
@@ -46,21 +55,29 @@ const rule = (primary, secondaryOptions) => {
 			return;
 		}
 
+		/** @type {Map<string, number>} */ const insideFunctions = new Map(
+			Object.entries(secondaryOptions?.insideFunctions ?? {}),
+		);
```

Comment:
[suggest] The following seems more consistent and readable a bit to me. Thoughts?

```suggestion
		/** @type {Map<string, number>} */
		const insideFunctions = new Map(Object.entries(secondaryOptions?.insideFunctions ?? {}));
```